examples/multi_window.py

```diff
@@ -7,6 +7,8 @@
 """
 
 def load_html():
+    import time
+    time.sleep(1)
```

Comment:
Ok, let's get rid of example timeouts for now and this PR can be merged. 

Also let's go with the original approach without get_key_instance(). Window management can be done via Javascript for now.